### DISCUSSION TITLE
General SBAS Interpretation Questions

### DISCUSSION THREAD

**Discussion by krkrueger** [gmtsar/gmtsar#745] [General]
Hey Everyone,

I am relatively new to the InSAR world and have now run a handful of real-world time series.  I have a few questions about the results.

1. I am struggling a little bit with the values.  My displacements seem to be really high. Looking at situations in the time series where the velocities are -350 mm/yr which seems very large.  I notice in a handful of other discussions and papers I have read that time series requires a reference point to a "quiet zone".  I don't have any reference points, and am not sure any areas that could be considered quiet there seems to be movement everywhere in the image.  Any other tips to getting reasonable values out of this without a solid reference point, or do I just present the results without units and explain that it is just relative changes with the rest of the measured area?  If I show this to people that do not have any InSAR experience, they look at the color bar and immediately discredit the process because of the large values, how would you explain this to someone quickly, that doesn't do InSAR? I am considering just leaving off the color bars values and doing a stretched color map with min/max values and label it as relative heave/subsidence.

2. Atmospheric correction.  I have played around with the -atm value, but there does not seem to be much difference in a large time series (2 years of images) if I use no atm or I use atm 3. There is a difference, so I know it is changing, but it is incredibly insignificant.  I look at the resulting aps_* files and there are certainly things in there, but I guess it doesn't really affect anything? 
 

3.  If I have these aps grids, can I use them on individual interferograms.  For instance if I have Int_T1_T2, and i have aps_T1, and aps_T2.  Would the corrected intf be Int_T1_T2_Corr = Int_T1_T2 +aps_T1-aps_T2? Or is this not the correct way to think about these things.

Thanks for your time!

**Comment by Xiaohua-Eric-Xu** [comment]
> 1. I don't have any reference points,

Even if you don't know any location being actually "quiet", selecting a reference will help reduce the variations in your time-series, most likely related to ambiguities of the unwrapped interferograms. Another way of course is to directly solve for the ambiguities, e.g., https://ieeexplore.ieee.org/abstract/document/8850313

> 1. I am considering just leaving off the color bars values and doing a stretched color map with min/max values and label it as relative heave/subsidence
 
Please don't do that. Reference your interferograms to a point and the solution will be more meaningful. 

> 2\. Atmospheric correction.  I have played around with the -atm value, but there does not seem to be much difference in a large time series (2 years of images) if I use no atm or I use atm 3.

This option affects the time-series (reducing scatters), but won't change velocity by a lot. 

> 3\. If I have these aps grids, can I use them on individual interferograms.

I believe so. In this paper, we actually used this type of approach to solve for APS first, apply that to interferograms, and then stacked the corrected interferograms, so as to resolve fault creep (tiny). https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2018GL080137

**Reply by krkrueger** [reply]
Awesome, thanks so much for the detailed response, very helpful!

**Reply by krkrueger** [reply]
Okay, attempting to implement these things, the strategy I am going to try to do is to take each unwrap.grd and make unwrap_corr.grd = unwrap_corr.grd - aps_T2.grd + aps_T1.grd.

Then going to pick a point in the hopefully quieter unwrap_corr.grds and make an unwrap_corr_pin.grd = unwrap_corr.grd - (ref_value).

Does this seem like a reasonable way to get roughly meaningful absolute values in both the corrected unwrap grids and the time series?  Let me know if there are any simple steps I am missing, or if I am off base on how I interpreted the help.

Thanks again!
